code/optimize/src/main/java/com/adobe/marketing/mobile/optimize/Offer.java

```diff
@@ -304,16 +317,35 @@ public void tapped() {
      * along with any additional XDM, free-form data, and override dataset identifier.
      *
      * @return {@code Map<String, Object>} containing the XDM data for the proposition interaction.
-     * @see OptimizeUtils#generateInteractionXdm(String, List, Boolean)
+     * @see OptimizeUtils#generateInteractionXdm(String, List)
      */
     public Map<String, Object> generateDisplayInteractionXdm() {
         if (propositionReference == null || propositionReference.get() == null) {
             return null;
         }
         return OptimizeUtils.generateInteractionXdm(
                 OptimizeConstants.JsonValues.EE_EVENT_TYPE_PROPOSITION_DISPLAY,
-                Collections.singletonList(propositionReference.get()),
-                true);
+                Collections.singletonList(propositionReference.get()));
+    }
+
+    /**
+     * Generates a map containing XDM formatted data for {@code Experience Event -
+     * OptimizeProposition Interactions} field group from this {@code OptimizeProposition} item.
```

Comment:
Same comment here. Also don't need @code for the Experience event part:
`...XDM formatted data for Experience Event OptimizeProposition Interactions field group from {@code Offer} list.`

Reply:
done @spoorthipujariadobe